src/components/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../Calculator';
+
+describe('Calculator component', () => {
+  it('renders correctly', () => {
+    const component = renderer.create(<Calculator />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Nicely done! Can you please do the same for all the components in your app according to the project requirement. 👍 

<img width="428" alt="Untitled" src="https://github.com/yusufobr/react-calculator/assets/103386183/0361b21f-e75f-4c4d-9e17-238151c6e3c7">